src/Fable.Transforms/Rust/Fable2Rust.fs

```diff
@@ -189,6 +189,9 @@ module TypeInfo =
     let makeArcTy com ctx (ty: Rust.Ty): Rust.Ty =
         [ty] |> makeImportType com ctx "Native" "Arc"
 
+    let makeBoxTy com ctx (ty: Rust.Ty): Rust.Ty =
+        [ty] |> makeImportType com ctx "Native" "Box"
```

Comment:
Please change to:
```fs
    let makeBoxTy com ctx (ty: Rust.Ty): Rust.Ty =
        [ty] |> makeImportType com ctx "Native" (rawIdent "Box")
```

Reply:
I have just tried this, but it does not work

```
error[E0433]: failed to resolve: use of undeclared type `Box_`
   --> tests\src\.\RecordTests.rs:387:17
    |
387 |                 Box_::from(Fable::Tests::RecordTests::BoxRecord { a: a.a + 1i32 })
    |                 ^^^^ use of undeclared type `Box_`

error[E0433]: failed to resolve: use of undeclared type `Box_`
   --> tests\src\.\RecordTests.rs:391:70
    |
391 |                 let actual: i32 = Fable::Tests::RecordTests::add1Box(Box_::from(
    |                                                                      ^^^^ use of undeclared type `Box_`
```

The issue seems to be that the filtering is happening further down within makeImportType

Reply:
@alexswan10k Then can you try this change?

```fs
    let getLibraryImportName (com: IRustCompiler) ctx moduleName typeName =
        let selector = moduleName + "_::" + (rawIdent typeName)
```

Reply:
Unfortunately this seems to create bigger problems downstream

```
error[E0252]: the name `arrayEmpty` is defined multiple times
  --> src\.\Array.rs:17:9
   |
16 |     use crate::module_5c9aa1da::Native_::arrayEmpty;
   |         ------------------------------------------- previous import of the value `arrayEmpty` here
17 |     use crate::module_5c9aa1da::Native_::arrayEmpty;
   |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ `arrayEmpty` reimported here
   |
   = note: `arrayEmpty` must be defined only once in the value namespace of this module

error[E0252]: the name `arrayEmpty` is defined multiple times
  --> src\.\List.rs:14:9
   |
13 |     use crate::module_5c9aa1da::Native_::arrayEmpty;
   |         ------------------------------------------- previous import of the value `arrayEmpty` here
14 |     use crate::module_5c9aa1da::Native_::arrayEmpty;
   |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ `arrayEmpty` reimported here
   |
   = note: `arrayEmpty` must be defined only once in the value namespace of this module
```

I don't really understand exactly how Rc and Arc are different. Are they not in the prelude?